lms/static/js/learner_dashboard/collections/program_collection.js

```diff
@@ -0,0 +1,8 @@
+(function (define) {
+  'use strict';
+  define(['backbone', 'js/learner_dashboard/models/program_model'], function (Backbone, Program) {
```

Comment:
Nitpick: I prefer the layout used here https://github.com/edx/programs/blob/master/programs/static/js/collections/course_runs_collection.js#L8 of a new line for each array entry (comment applies for all define functions in this PR)